docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,66 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+Kubernetes allows configuring and managing resources for containers. With REANA, you can configure cluster wide **memory** limits to make the effiecient use of available resources.
+
+To set the memory limit of a job you must specify `kubernetes_memory_limit`.
+
+This can be set directly in `bytes` and you can read more about the expected memory values on [kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
+
+Note that jobs exceeding the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits).
```

Comment:
The approach should be the opposite. As a user, by default, you don't touch anything, and if your job is so heavy that you get `OOMKilled`, then you have the possibility to increase the memory limit to avoid this error.

Reply:
Then you can also mention that one can also decrease the memory limit to increase the chances of their jobs being scheduled earlier.